app/src/test/kotlin/com/intive/picover/profile/viewmodel/ProfileViewModelTest.kt

```diff
@@ -80,5 +81,31 @@ class ProfileViewModelTest : ShouldSpec(
 
 			tested.profile.value shouldBe Loading
 		}
+
```

Comment:
missing test for setting userName when fetching profile :) this can be added using assertSoftly for loaded test

Reply:
Done